ci/gdk_build.template.steps.yaml

```diff
@@ -19,7 +19,7 @@ windows: &windows
     - "agent_count=1"
     - "capable_of_building=gdk-for-unreal"
     - "environment=production"
-    - "machine_type=quad-high-cpu"  # this name matches to SpatialOS node-size names
+    - "machine_type=${MACHINE_TYPE_ENVVAR}"
```

Comment:
Nit: It seems to me that dropping the `_ENVVAR` would be more in line with the rest of the code w.r.t. variable naming

Reply:
Yeah, also wasn't sure what to name this, but I was worried about `MACHINE_TYPE` interfering with something else
Maybe some other suffix or a completely different name?

Reply:
`GCP_MACHINE_TYPE`? `BK_MACHINE_TYPE`?

Reply:
BK_MACHINE_TYPE sounds good to me